__tests__/Unit/Components/Tasks/Card.test.tsx

```diff
@@ -231,6 +233,60 @@ describe('Task card', () => {
         expect(closeTaskBtn).toBeInTheDocument();
     });
 
+    it('Should make PATCH call with feature flag', function () {
+        // const mockFetch = jest.mock('fetch');
+        function Wrapper({
+            children,
+        }: PropsWithChildren<Record<string, never>>): JSX.Element {
+            return <Provider store={store()}>{children}</Provider>;
+        }
+        const { result } = renderHook(() => useUpdateTaskMutation(), {
+            wrapper: Wrapper,
+        });
+        const [updateTask] = result.current;
+
+        const PROPS = {
+            ...DEFAULT_PROPS,
+            content: {
+                ...DEFAULT_PROPS.content,
+                status: 'Available',
+                assignee: undefined,
+                github: {
+                    issue: {
+                        assignee: 'johndoe',
+                        status: 'open',
+                        id: 12278,
+                        assigneeRdsInfo: {
+                            username: 'john',
+                            firstName: 'John',
+                            lastName: 'Doe',
+                        },
+                    },
```

Comment:
please import this from __mocks__/db folder and use here as constant

Reply:
I am not longer making any changes in this file.